app/grandchallenge/codebuild/client.py

```diff
@@ -0,0 +1,85 @@
+from time import sleep
+
+import boto3
+from django.conf import settings
+from django.core import files
+from django.core.files.temp import NamedTemporaryFile
+
+from grandchallenge.algorithms.models import AlgorithmImage
+
+
+class CodeBuildClient:
+    build_id = None
+
+    def __init__(
+        self, *, project_name,
+    ):
+        self.client = boto3.client(
+            "codebuild",
+            aws_access_key_id=settings.CODEBUILD_ACCESS_KEY,
+            aws_secret_access_key=settings.CODEBUILD_SECRET_KEY,
+            region_name="eu-central-1",
+        )
+        self.s3_client = boto3.client(
```

Comment:
You shouldn't need to set up the separate s3 client, can't you use `grandchallenge.core.storage.private_s3_storage` which is already configured?

Reply:
I'll look into this.